Scrapper-Service/database/mdb.py

```diff
@@ -56,15 +56,37 @@ def __del__(self):
         self.logger.info("Closing connection to mongodb !!")
         self.client.close()
         self.logger.info("Succesfully Closed connection to mongodb !!")
-
+        
+    def getDateRange(self,_id):
+        data = self.collection.find({'_id':_id})
+        date = None 
+        for dt in data:
+            date = dt['dateRange']
```

Comment:
* Small thing but variable naming should reflect intend. date_range is a better name